source/blog/tags.html

```diff
@@ -8,6 +8,17 @@
 
 <div>
 {% for tag,posts in data.posts_tags %}
-<a href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
+
+<!-- conditions to hide author tag names on the tags page --> 
+  {% if tag == "nikki-stevens" %} 
+    <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a> 
+  {% elseif tag == "mike-klass" %}
+     <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a> 
+  {% elseif tag == "jason-savino" %}
+     <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
+ 
+  {% else %} 
+  <a href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
```

Comment:
Ahhh, I only focused on the actual tags page from the Tags menu selection
up top.  I forgot about the articles themselves

On Tue, Aug 18, 2015 at 9:06 AM, Nikki Stevens <notifications@github.com>
wrote:

> In source/blog/tags.html
> <https://github.com/waybetterdev/dev.waybetter.com/pull/10#discussion_r37317746>
> :
>
> > @@ -8,6 +8,17 @@
> >
> >  <div>
> >  {% for tag,posts in data.posts_tags %}
> > -<a href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
> > +
> > +<!-- conditions to hide author tag names on the tags page -->
> > +  {% if tag == "nikki-stevens" %}
> > +    <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
> > +  {% elseif tag == "mike-klass" %}
> > +     <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
> > +  {% elseif tag == "jason-savino" %}
> > +     <a class="hide" href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
> > +
> > +  {% else %}
> > +  <a href="{{ site.url }}/blog/tags/{{ tag|url_encode(true) }}">{{ tag }}</a>
>
> ALSO, I'm just now testing this and it doesn't seem to work. :(
> https://dl.dropboxusercontent.com/s/l7h0p77g6c7ymsm/2015-08-18%20at%209.06%20AM%202x.png?dl=0
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/waybetterdev/dev.waybetter.com/pull/10/files#r37317746>
> .
>



-- 
Mahalo,


Teresa Nededog | Full Stack Software Engineer
804.252.7447 | Github <http://github.com/treindle> | Portfolio
<http://portfolio.reindle.io/>